Source/Core/Core/Boot/Boot.h

```diff
@@ -5,13 +5,16 @@
 #pragma once
 
 #include <cstdlib>
+#include <optional>
 #include <string>
+#include <variant>
 
 #include "Common/CommonTypes.h"
+#include "DiscIO/Enums.h"
+#include "DiscIO/Volume.h"
 
 namespace DiscIO
 {
-class IVolume;
```

Comment:
I remember forward declaring DiscIO::IVolume used to cause build issues. I've reverted that change and pushed the commit. Let's see if this builds now.